spec/index.md

```diff
@@ -5462,7 +5462,7 @@ Q.canister_id = ic_principal
 Q.method_name = 'fetch_logs'
 Q.arg = candid(A)
 A.canister_id = effective_canister_id
-S[A.canister_id].canister_log_visibility = Public or M.caller ∈ S.controllers[A.canister_id]
+S[A.canister_id].canister_log_visibility = M.caller ∈ S.controllers[A.canister_id] or Public
```

Comment:
why did you change this line? it looks weird now?

Reply:
first, I thought that oder might matter (I was not sure if it was) and the most left value should be considered as the default one, that's why I moved to have controllers first and public the second.

and secondly I realised that `log_visibility` is an enum, not the actual controllers list so changed to `Controllers or Public`.